app/controllers/concerns/decidim/civicrm/admin/needs_multiselect_snippets.rb

```diff
@@ -18,7 +18,7 @@ def snippets
           unless @snippets.any?(:select2)
             @snippets.add(:select2, ActionController::Base.helpers.stylesheet_link_tag("https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/css/select2.min.css"))
             @snippets.add(:select2, ActionController::Base.helpers.javascript_include_tag("https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/js/select2.min.js"))
-            @snippets.add(:select2, ActionController::Base.helpers.javascript_pack_tag("decidim_admin_civicrm_selects"))
+            # @snippets.add(:select2, ActionController::Base.helpers.javascript_pack_tag("decidim_admin_civicrm_selects"))
             @snippets.add(:select2, ActionController::Base.helpers.stylesheet_pack_tag("decidim_admin_civicrm_selects"))
             @snippets.add(:head, @snippets.for(:select2))
           end
```

Comment:
Select2 is no longer necessary in Decidim 28.
Now we use [TomSelect](https://tom-select.js.org/docs/).
An example of implementation:
https://github.com/decidim-ice/decidim-module-decidim_awesome/blob/84374037d34a3ac80dc18406834169c65869f11b/app/packs/src/decidim/decidim_awesome/admin/proposal_sortings.js#L3

This means that we no longer need these snippets and we need to rebuild the javascript using it